samples/house_price_estimates/README.md

```diff
@@ -89,7 +116,13 @@ Click on each step in the Kubeflow Pipeline GUI and view its log to see the outp
 * train-model runs trains the machine learning model and then runs it on the test data, printing out a sample of the results
 * submit-result prints out the data catalog asset ID of the newly created asset containing the results
 
-See slides 21-24 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing).
+See slides 21-24 of the [demo presentation](https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing) for the intermediate steps.
+
+The final state with submit-result step results will appear as follows:
+<br><center><img src="./img/kfp_gui/15_view_run_result.jpeg" width=600></center>
+
+After the run result is complete you can go to the catalog and see the new catalog entry, and in your storage you can see the newly created result data.
```

Comment:
Not sure what you mean by `run result`. Should it be just `run`?

Reply:
Fixed